src/ui/popup/components/header/style.less

```diff
@@ -45,7 +45,7 @@
             display: inline-block;
             height: @size-control-inner + 2 * @size-border;
             left: 100%;
-            opacity: 0;
+            opacity: 0.7;
```

Comment:
made this one to be visible a bit because it's very hard to notice with zero opacity